docs/source/tutorial.rst

```diff
@@ -293,11 +293,56 @@ This should now return without any message because all operations have already b
 
     To simply, execute a specific operation from the command line ignoring all logic, use the ``exec`` command, *e.g.*: ``$ python project.py exec compute_volume``.
     This command (as well as the run command) also accepts jobs as arguments, so you can specify that you only want to run operations for a specific set of jobs.
+    
+    
+Define groups of job-operations
+-------------------------------
+
+It can be more convenient to submit several operations once instead of waiting for previous operation doone and submit new operation manually. For instance,
```

Comment:
Small typo here `done`
```suggestion
It can be more convenient to submit several operations once instead of waiting for previous operation done and submit new operation manually. For instance,
```